<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProceedDialog</class>
 <widget class="QDialog" name="ProceedDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proceed objects</string>
  </property>
  <property name="windowIcon">
   <iconset theme="system-run">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="distanceLabel">
     <property name="text">
      <string>Maximum label distance</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="distanceSpin">
     <property name="specialValueText">
      <string>Unlimited label distance</string>
     </property>
     <property name="suffix">
      <string> m</string>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lineLabel">
     <property name="text">
      <string>Linear objects strategy</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="lineCombo">
     <item>
      <property name="text">
       <string>Skip closed lines</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Keep closed lines</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="pointLabel">
     <property name="text">
      <string>Point objects strategy</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="pointCombo">
     <item>
      <property name="text">
       <string>Skip text without symbol</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Insert symbol to text</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="symbolLabel">
     <property name="text">
      <string>Point objects symbol</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="symbolEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProceedDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProceedDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pointCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ProceedDialog</receiver>
   <slot>pointStrategyChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>693</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pointStrategyChanged(int)</slot>
 </slots>
</ui>
